### PR TITLE
website / workspace layouts

### PR DESCRIPTION
**Pull request by emielvdveen**

### CODE REVIEW

File: core/assets/js/app.js [hunk 1]
```diff
@@ -118,19 +130,20 @@ Hooks.PythonUploader = {
...
-  params: {
-    _csrf_token: csrfToken,
-  },
+
+let liveSocket = new LiveSocket('/live', Socket, {
   dom: {
     onBeforeElUpdated(from, to) {
       if (from.__x) {
-        Alpine.clone(from.__x, to);
+        window.Alpine.clone(from.__x, to)
```
Comment (bliksemman) [inline core/assets/js/app.js:138]: Why this change?

File: core/assets/js/app.js [hunk 2]
```diff
@@ -57,6 +57,18 @@ let csrfToken = document
   .querySelector("meta[name='csrf-token']")
   .getAttribute("content");
 let Hooks = {};
+
+Hooks.NativeWrapper = {
+  mounted() {
+    window.nativeWrapperHook = this
```
Comment (bliksemman) [inline core/assets/js/app.js:63]: I don't see the usage of `nativeWrapperHook`. Am I overlooking something?

File: core/assets/static/images/icons/signout_active.svg [hunk 3]
```diff
@@ -0,0 +1,88 @@
+%PDF-1.7
```
Comment (bliksemman) [inline core/assets/static/images/icons/signout_active.svg:1]: This seems to be PDF, so not an svg (like the filename implies). Is there a reason for this?

**Review by bliksemman — APPROVED** [review]

File: core/assets/js/app.js (hunk 1 above)
Comment (emielvdveen) [reply core/assets/js/app.js:138]: A left over from some changes I tried to fix an Alpine data state problem of first time after load. I think it is safe to remove

File: core/assets/js/app.js (hunk 2 above)
Comment (emielvdveen) [reply core/assets/js/app.js:63]: This is in EyraUI.Navigation.Button

```
      <div id={{@id}} phx-hook="NativeWrapper" @click="nativeWrapperHook.toggleSidePanel(); {{ @overlay }}" >
        <Get :if={{ @method === :get && !is_dead?(@action)}} path={{@action.target}}>
```

File: core/assets/static/images/icons/signout_active.svg (hunk 3 above)
Comment (emielvdveen) [reply core/assets/static/images/icons/signout_active.svg:1]: Oops.. will change it

File: frameworks/eyra_ui/lib/components/navigation/button/get.ex [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+
+  prop(path, :string, required: true)
+
+  slot(default, required: true)
+
+  def render(assigns) do
+    ~H"""
+      <a
+        class="cursor-pointer"
+        x-data="{}"
```
[the remark this answers is not in the document]
Comment (emielvdveen) [reply frameworks/eyra_ui/lib/components/navigation/button/get.ex:12]: Only needed to create Alpine component, not needed here. Will remove it
